mPowerSDK/Startup/APHAppDelegate.m

```diff
@@ -266,10 +266,22 @@ - (void)setUpAppAppearance
     [APCAppearanceInfo setAppearanceDictionary:self.appearanceInfo];
     [[UINavigationBar appearance] setTintColor:[UIColor appPrimaryColor]];
     [[UINavigationBar appearance] setTitleTextAttributes: @{
-                                                            NSForegroundColorAttributeName : [UIColor appSecondaryColor2],
+                                                            NSForegroundColorAttributeName : [UIColor appPrimaryColor],
                                                             NSFontAttributeName : [UIFont appNavBarTitleFont]
                                                             }];
     [[UIView appearance] setTintColor:[UIColor appPrimaryColor]];
+    
+    // TAB BAR APPEARANCE VALUES
+    
+    // Background of tab bar cell
+    [[UITabBar appearance] setBarTintColor: [UIColor whiteColor]];
+    // Select tint of tab icon
+    [[UITabBar appearance] setTintColor: [UIColor appPrimaryColor]];
+    // Color of un-selected tab icon is red:142.0f/255.0f green:142.0f/255.0f blue:149.0f/255.0f
+    [[UITabBarItem appearance] setTitleTextAttributes:[NSDictionary dictionaryWithObjectsAndKeys:[UIColor colorWithRed:142.0f/255.0f green:142.0f/255.0f blue:149.0f/255.0f alpha:1.0f], NSForegroundColorAttributeName, nil]
```

Comment:
Sure thing.  I hate defining a bunch of grey colors, but I'll see if an existing one is close enough